components/eam/bld/namelist_files/use_cases/1950_eam_CMIP6_chemUCI-Linoz-mam5-vbs.xml

```diff
@@ -0,0 +1,136 @@
+----- modified based on 1850 version ---- combined with 1950 use_case for V2
+
+<?xml version="1.0"?>
+<namelist_defaults>
+
+<!-- Set default output options for CMIP6 simulations -->
+<cosp_lite>.true.</cosp_lite>
+
+<!-- Atmosphere initial condition -->
+<ncdata dyn="se" hgrid="ne30np4" nlev="72" >atm/cam/inic/homme/NGD_v3atm.ne30pg2.eam.i.0001-01-01-00000.c20230106.nc</ncdata>
+<!--ncdata dyn="se" hgrid="ne0np4_northamericax4v1" nlev="72" >atm/cam/inic/homme/cami_0001-01-01_northamericax4v1_c190709.nc</ncdata -->
+
+<!-- Solar constant from CMIP6 input4MIPS -->
+<solar_data_file>atm/cam/solar/Solar_1950control_input4MIPS_c20171208.nc</solar_data_file>
+<solar_data_ymd>19500101</solar_data_ymd>
+<solar_data_type>FIXED</solar_data_type>
+
+<!-- 1950 GHG values from CMIP6 input4MIPS -->
+<!-- <co2vmr>284.317e-6</co2vmr> The CMIP6 concentration set by CCSM_CO2_PPMV in cime/src/drivers/mct/cime_config/config_component_acme.xml -->
+<ch4vmr>1163.821e-9</ch4vmr>
+<n2ovmr>289.739e-9</n2ovmr>
+<f11vmr>62.83147e-12</f11vmr>
+<f12vmr>6.382257e-12</f12vmr>
+
+<!-- For comprehensive history -->
+<history_amwg       >.true.</history_amwg>
+<history_aerosol    >.true.</history_aerosol>
+<history_aero_optics>.true.</history_aero_optics>
+
+<!-- External forcing for BAM or MAM.  CMIP6 input4mips data -->
+<ext_frc_type         >CYCLICAL</ext_frc_type>
+<ext_frc_cycle_yr     >1950</ext_frc_cycle_yr>
+<no2_ext_file         >atm/cam/chem/trop_mozart_aero/emis/chem_gases/2degrees/emissions-cmip6_NO2_aircraft_vertical_1750-2015_1.9x2.5_c20170608.nc </no2_ext_file>
+<!--  Use 1850-2014 averaged volcanic SO2 emission for 1950 -->
+<so2_ext_file         >atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_1850_2014_avg_so2_elev_strat_1950_c20230310.nc </so2_ext_file>
```

Comment:
@wlin7 It may need some clarifications. SO2 external forcing contains several sectors of emission, volcanic emission is only a part of it. In this file, ideally, it should be 1950s emissions from other sectors in addition to 1850-2014 averaged volcanic emission.

Reply:
Thanks for carefully reviewing the settings, @keziming .  In v2, the SO2 external forcing for 1950 compset appears to be  cycling over a single year 1950 instead of an average. (file `atm/cam/chem/trop_mozart_aero/emis/DECK_ne120/cmip6_mam4_so2_elev_1950-1960_c180203.nc`). We can also use single year 1950 or an average around 1950. Can you please help to update those other sectors? We should have some discussion on 1950 forcings in general in next atm group meeting.